spec/helpers/chat_mirgation_message_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe ChatMigrationMessage, type: :helper do
+  describe '#display_message?' do
+
+    context 'before display_message date' do
+      it 'returns false' do
+      	allow(Date).to receive(:today) { Date.new(2016, 2, 2) }
+        expect(display_chat_migration_message?).to be false
+      end
+    end
+
+    context 'on display_message date' do
+      it 'returns true' do
+        allow(Date).to receive(:today) { Date.new(2016, 2, 6) }
+        expect(display_chat_migration_message?).to be true
+       end
+     end
+
+     context 'after display_message date' do
+       it 'returns false' do
+         allow(Date).to receive(:today) { Date.new(2016, 2, 10) }
+         expect(display_chat_migration_message?).to be false
+       end
+     end
+  end
+end
```

Comment:
Hi @JiggyPete this is a good point, the test is essentially for anything before and anything after, so testing the day directly before/after makes more sense than a few days before/after, I've commited the change